item.rb

```diff
@@ -0,0 +1,22 @@
+class Item
+  attr_accessor :publish_date
+  attr_reader :archived, :id, :genre, :label, :author
+
+  def initialize(publish_date, archived: false, id: nill)
```

Comment:
Hello, @daudi13 this is okay but when data is saved, it has an id so you can set it as optional but have it as `id: rand(1..10_000)`

Reply:
Thanks for pointing that out @iLynette